shared/services/rewards/types.go

```diff
@@ -14,6 +14,8 @@ import (
 type rewardsFileVersion uint64
 
 const (
+	FarEpoch uint64 = 18446744073709551615
```

Comment:
generator-impl-v2 previously defined this, but it is used in several implementations.

Reply:
... and it has to be in a separate `const` block from the enum, or it takes a position